### DISCUSSION TITLE
communication error on the web server

### DISCUSSION THREAD

**Discussion by kluaneit** [glpi-project/glpi-agent#452] [Q&A]
### Bug reporting acknowledgment

Yes, I read it

### Professional support

Yes, I know

### Describe the bug

I have a problem when I did a domain migration to another hosting, in the old hosting the GLPI worked great but when migrating to the new one it no longer works for me and I did all the clean installation with the restoration of the database and I couldn't get the Agent to communicate with the Web Server

### To reproduce
```
[Tue Jul 25 10:56:04 2023][info] [http client] SSL server certificate fingerprint: sha256$608569b3030e68168f6a6c20fee06eae9005ef0f72c4e71fb658e73fe73524f6
[Tue Jul 25 10:56:04 2023][info] [http client] You can set it in conf as 'ssl-fingerprint' and disable 'no-ssl-check' option to trust that server certificate
[Tue Jul 25 10:56:04 2023][error] [http client] communication error: 406 Not Acceptable
[Tue Jul 25 10:56:04 2023][error] No supported answer from server at https://inventario.kluanenicaragua.com/glpi/front/inventory.php
[Tue Jul 25 10:56:04 2023][info] target server0: next run: Tue Jul 25 11:04:04 2023 - https://inventario.kluanenicaragua.com/glpi/front/inventory.php
[Tue Jul 25 10:56:04 2023][info] GLPI Agent memory usage: WSS=3346432 PFU=128876544
```
### Expected behavior

to register on the web server

### Operating system

Windows

### GLPI Agent version

1.4

### GLPI version

10.0.x (See additional context below)

### GLPIInventory plugin or FusionInventory for GLPI plugin version

GLPI Inventory v1.0.x (See additional context below)

**Comment by g-bougard** [comment]
Hi @kluaneit 

first, as you didn't obfuscate you glpi nurl, and in the case you're using a public glpi, be sure to use the latest version to include all latest security fixes. The currently latest version in 10.0.9.

Then I don't see here any evidence of glpi-agent bug. A 406 error is probably related to a server configuration issue involved by the migration. You should investigate your http server configration.

Regarding your glpi-agent `server` parameter configuration, be sure to read this documentation: https://glpi-agent.readthedocs.io/en/latest/configuration.html#server

About SSL support, try to not use `no-ssl-check` in production by eventually using `ssl-fingerprint` with the value reported in your log, or you can deploy the server certificate using `ca-cert-file` on agent side or deploy it in the system key store on windows.

Also don't forget to update to latext glpi-agent v1.5 version.
